spec/lib/pipedrive/util_spec.rb

```diff
@@ -0,0 +1,22 @@
+# frozen_string_literal: true
+
+require "spec_helper"
+
+RSpec.describe Pipedrive::Util do
+  describe "#raise_error" do
+    context "404 status" do
```

Comment:
I know that you are covering `404` and then an `else`, but I'm curious why and if it could be necessary for the future to cover other errors such as 500 for instance.

Reply:
It is incomplete and experimental, it is just to support exceptions as part of flow control.

TODO: Go thru the list of errors returned by Pipedrive.